website/docs/r/recovery_services_vault_guard_proxy.html.markdown

```diff
@@ -0,0 +1,74 @@
+---
+subcategory: "Recovery Services"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_recovery_services_vault_guard_proxy"
+description: |-
+  Manages a Recovery Services Vault Guard Proxy.
+---
+
+# azurerm_recovery_services_vault_guard_proxy
+
+Manages a Recovery Services Vault Guard Proxy.
```

Comment:
whaat his this actually doing?

should this be "manages an assoication of a resource guard and vault?"  or "adds a resource guard proxy to a vault?"